.gitignore

```diff
@@ -1,3 +1,6 @@
+*.csv
+*.csv.gz
```

Comment:
The `rapids-sagemaker-higgs` notebook has some steps that `wget` and then `gunzip` the Higgs dataset, which leaves a big `.csv` and `.csv.gz` around.

Adding this to ensure `git` ignores them.

(this change brought to you by me running `git add ./examples` and wondering why it was taking over a minute to run 😅 )